examples/secured-service-with-basic-auth/secured_service_with_basic_auth.bal

```diff
@@ -43,8 +43,8 @@ service<http:Service> echo bind ep {
     }
     // The authentication and authorization settings can be overridden at
     // resource level.
-    // The hello resource would inherit the authentication:{enabled:true} flag
-    // from the service level, and override scope defined in service level
+    // The hello resource would inherit the `authentication:{enabled:true}` flag
+    // from the service level, and override the scope defined in service level
```

Comment:
Can you double check with the original author what the following phrase means: "scope defined in service level (xxx) with scope2".

Reply:
The service level scope, which was xxx previously, has been modified to scope1. Hence, the correct phrase would be:
"scope defined in service level (scope1) with scope2".

Reply:
Fixed https://github.com/ballerina-platform/ballerina-examples/pull/767/commits/5ba58e0583097d0a83e1a10eecda3b2584fea46d